CHANGES

```diff
@@ -10,6 +10,8 @@ Incompatible changes
 * #9222: Update Underscore.js to 1.13.1
 * #9217: manpage: Stop creating a section directory on build manpage by default
   (see :confval:`man_make_section_directory`)
+* #9240: Unknown node error for pending_xref_condition is raised if an extension
```

Comment:
@tk0miya I just noticed, that you put this in the `Incompatible changes` section instead of `Bugfixes`. Also, this should probably read "Unknown node error for pending_xref_condition is _**no longer**_ raised if ...".

Reply:
Oops, it's my wrong...